### PR TITLE
Task/order journey

### PR DESCRIPTION
**Pull request by MuhammadHamza096**
### GOAL
implement order journey

### WORK
integrate order api 

### RISK
N/A

### CODE REVIEW

File: src/Features/Customer/Dashboard/index.js [hunk 1]
```diff
@@ -1,24 +1,89 @@
...
+      type: AUTO_COMPLETE,
+      label: '',
+      values: [],
+      placeholder: 'Select Vendor ',
+      value: '',
+      isValid: true,
+      errorMessage: '',
+
+      onChange: (event, value) => {
+        if (value !== null) {
```
Comment (ShahirZain) [inline src/Features/Customer/Dashboard/index.js:35]: we can directly check value in if condition

**Review by ShahirZain — APPROVED** [review]

File: src/Features/Customer/Dashboard/index.js [hunk 2]
```diff
@@ -1,24 +1,92 @@
...
 
 import MainTab from '../../../components/CardMenus/Tabs';
+import AddEditForm from '../../../components/CommonGridBasedForm';
+import { AUTO_COMPLETE } from '../../../components/CommonGridBasedForm/FieldTypes';
 import TemporaryDrawer from '../../../components/Drawer';
-import { openDrawer } from '../actions';
+import Loader from '../../../components/Loader';
+import NoDataFilter from '../../../components/NoDataFilter';
+import { SelectChangeHandler } from '../../../util/CommonGridBasedFormUtils';
+import { cardData } from '../actions';
```
Comment (SiddiquiMonis401) [inline src/Features/Customer/Dashboard/index.js:14]: What is this cardData means in actions if you are getting card data it should be like getCardData

File: src/components/NoDataFilter/index.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import NoData from '../../assets/no-data.jpg';
+import { NoDataContainer, ImageContainer, PlaceHolder } from './style';
+
+function NoDataFilter({ text }) {
```
Comment (SiddiquiMonis401) [inline src/components/NoDataFilter/index.js:6]: Shouldn't it be like EmptyData or something more meaningful beacause what i thing after seeing this is not telling me what this component is actually doing.

File: src/components/Drawer/index.js [hunk 4]
```diff
@@ -35,9 +36,16 @@ import {
   CartPaper,
   ModalDiv,
 } from './style';
-const TemporaryDrawer = () => {
+const TemporaryDrawer = ({ mutate }) => {
```
Comment (SiddiquiMonis401) [inline src/components/Drawer/index.js:39]: What does temporary drawer means ??

File: src/Features/Customer/actions.js [hunk 5]
```diff
@@ -64,3 +65,9 @@ export const editUser = (data) => {
     payload: data,
   };
 };
+export const cardData = (data) => {
```
Comment (SiddiquiMonis401) [inline src/Features/Customer/actions.js:68]: It should be getCartData

File: src/Features/Customer/Dashboard/style.js [hunk 6]
```diff
@@ -13,7 +13,11 @@ export const VendorGrid = styled(Grid)`
   }
   `};
 `;
+
 // export default VendorGrid;
```
Comment (SiddiquiMonis401) [inline src/Features/Customer/Dashboard/style.js:17]: remove commented code

**Review by SiddiquiMonis401 — CHANGES_REQUESTED** [review]

**Review by SiddiquiMonis401 — APPROVED** [review]

**Pull request merged**